doc/source/reference/arrays.datetime.rst

```diff
@@ -119,6 +119,9 @@ simple datetime calculations.
     >>> np.timedelta64(1,'W') / np.timedelta64(1,'D')
     7.0
 
+    >>> np.timedelta64(1,'W') % np.timedelta64(10,'D')
+    numpy.timedelta64(7,'D')
```

Comment:
Shouldn't this be 4?

Reply:
Why? 10 days goes into a week zero times--I copy pasted your suggested example, and that's the answer I got on this feature branch. Seems consistent with `small_operand % larger_operand` more broadly producing `small_operand`, no?

Reply:
You're right, I'm wrong here